README.md

```diff
@@ -1 +1,58 @@
-# HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+ Description of the project: 
+    This project represents the mobile version and the Desktop version of the first capstone project from Microverse, Where I used all the skills and knowledge that I acquired form the first module.
+
+
+## Built With
+
+- Major languages: HTML
+-  CSS
+    - Flexbox
+    - CSS Grid
+- JavaScript
+- Mobile first aproach
+- linter for code analysis
+
```

Comment:
We should not jump over sections from the README template. Always we need to give clear instructions to users to clone our repo.

- [ ] Kindly, I ask you to implement back all the sections from the README template and provide clear instructions for cloning the repo.
The link to the README template is [here](https://github.com/microverseinc/readme-template).